ui/package.json

```diff
@@ -23,22 +23,24 @@
   },
   "dependencies": {
     "@babel/preset-typescript": "^7.12.7",
-    "@material-ui/core": "4.11.1",
+    "@material-ui/core": "^4.11.3",
     "@material-ui/icons": "^4.0.0",
     "@material-ui/lab": "^4.0.0-alpha.56",
     "@material-ui/styles": "^4.11.3",
     "leaflet": "^1.0.2",
     "leaflet.markercluster": "^1.4.1",
     "material-ui-dropzone": "^3.5.0",
-    "mui-virtualized-table": "3.0.0-6",
+    "mui-virtualized-table": "^2.2.3",
```

Comment:
Please bump it up to` "mui-virtualized-table": "3.0.0-6" `before merge, the older version has some bugs that are incompatible with our other MUI components